modules/rubiconBidAdapter.js

```diff
@@ -110,58 +110,82 @@ export const spec = {
     const videoRequests = bidRequests.filter(bidRequest => bidType(bidRequest) === 'video').map(bidRequest => {
       bidRequest.startTime = new Date().getTime();
 
-      let params = bidRequest.params;
-      let size = parseSizes(bidRequest);
-
-      let data = {
-        page_url: _getPageUrl(bidRequest, bidderRequest),
-        resolution: _getScreenResolution(),
-        account_id: params.accountId,
-        integration: INTEGRATION,
-        'x_source.tid': bidRequest.transactionId,
-        timeout: bidderRequest.timeout - (Date.now() - bidderRequest.auctionStart + TIMEOUT_BUFFER),
-        stash_creatives: true,
-        slots: []
+      const data = {
+        id: bidRequest.transactionId,
+        test: config.getConfig('debug') ? 1 : 0,
+        cur: ['USD'],
+        source: {
+          tid: bidRequest.transactionId
+        },
+        tmax: config.getConfig('TTL') || 1000,
```

Comment:
tmax vs ttl.

TMAX is how long PBS gets to run its auction before it stops and responds (more like how long PBS gives the bidders)

TTL if supposed to be how long the bid is available / cached for I think.

Need to find out what config.getConfig is getting for TTL and what its setting is supposed to be.